pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "8.1.0-dev"
+	bundleVersion        = "8.3.0-dev"
```

Comment:
Current release has `8.0.2` so I think this should stay as `8.1.0-dev`.

https://github.com/giantswarm/releases/blob/d8278f3413dad2bd0ca5073ddc4c896dbf8dd0fa/aws.yaml#L6

When we cut a new release I think it should be `8.0.3` or `8.1.0`.

Reply:
ack